netplugin/agent/docker_event.go

```diff
@@ -31,9 +32,20 @@ import (
 	"golang.org/x/net/context"
 )
 
+func (ag *Agent) lbServiceExist() bool {
+	lbCfg := &mastercfg.CfgServiceLBState{}
+	plugin := ag.Plugin()
+	lbCfg.StateDriver = plugin.StateDriver
+
+	if lbList, err := lbCfg.ReadAll(); err == nil {
+		return len(lbList) > 0
+	}
+	return false
```

Comment:
its is ok here.
```readAll()``` takes care of retry/cluster unavailable 
and there is no point in proceeding in case of error or key not found